jasmin_cloud/provider/dto.py

```diff
@@ -75,7 +75,7 @@ class Size(namedtuple('Size', ['id', 'name', 'cpus', 'ram', 'disk'])):
 class Machine(namedtuple('Machine', ['id', 'name', 'image', 'size',
                                      'status', 'power_state', 'task',
                                      'internal_ip', 'external_ip', 'nat_allowed',
-                                     'attached_volume_ids', 'owner', 'created'])):
+                                     'attached_volume_ids', 'owner', 'created', 'provisioned_by_caas'])):
```

Comment:
This new field is not populated anywhere.

`nat_allowed` is also populated from machine metadata, so use that as an example.